index.ts

```diff
@@ -118,6 +119,34 @@ function cancelTriggers() {
     });
 }
 
+/**
+ * Runs a one-off script on the sheet.
+ *
+ * This may be helpful for a quick hack to make a change on a big sheet, and it
+ * can be run either from the DEBUG menu in the sheet itself or the
+ * script.google.com editor.
+ */
+function oneOffScript() {
```

Comment:
Instead of calling it oneOffScript, I think it's better to name it specifically about what it actually does.
So it's much easier for user to understand what will happen if they click on it.
Also update the doc.